src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -311,14 +311,15 @@ public static string MachineName
 
         public static string NewLine => "\n";
 
-        private static Lazy<OperatingSystem> s_osVersion = new Lazy<OperatingSystem>(() =>
+        private static Lazy<OperatingSystem> s_osVersion = new Lazy<OperatingSystem>(GetOperatingSystem(Interop.Sys.GetUnixRelease()));
```

Comment:
This isn't actually lazy anymore... either it should be changing to be a `private static readonly OperatingSystem s_osVersion = GetOperatingSystem(Interop.Sys.GetUnixRelease());`, or the lambda should be restored.

Reply:
https://github.com/dotnet/corefx/pull/31332#discussion_r205486461

https://github.com/dotnet/corefx/pull/31702